Antidote/CallsManager.m

```diff
@@ -222,6 +222,23 @@ - (void)updateCurrentCallInterface:(OCTCall *)call
         else {
             activeVC.callDuration = call.callDuration;
         }
+
+        if (call.friendSendingVideo && ! activeVC.videoView) {
```

Comment:
No need for this check `! activeVC.videoView`. Controller will handle case of adding same `videoView`